ledger/testchain-generator/src/main.rs

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2019-2025 Provable Inc.
+// This file is part of the snarkVM library.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use snarkvm_console::prelude::{CanaryV0, MainnetV0, Network, TestRng, TestnetV0};
+use snarkvm_ledger::{Ledger, store::helpers::rocksdb::ConsensusDB, test_helpers::TestChainBuilder};
+
+use aleo_std::StorageMode;
+
+use anyhow::{Context, Result, bail};
+use clap::{Parser, builder::PossibleValuesParser};
+
+#[derive(Parser)]
+struct Args {
+    /// The number of validators active on the chain.
+    /// (also corresponds to the number of certificates per round)
+    num_validators: usize,
+    /// The number of blocks to generator.
+    num_blocks: usize,
+    /// Load the genesis block for the chain from the specified path instead of generating it.
+    #[clap(long)]
+    genesis_path: Option<String>,
+    /// Set a custom storage path for the ledger.
+    /// By default, it will use the ledger of the first devnet validator.
+    #[clap(long, short = 'p')]
+    storage_path: Option<String>,
+    /// Remove existing ledger if it already exists.
+    #[clap(long, short = 'f')]
+    force: bool,
```

Comment:
```suggestion
    overwrite_storage: bool,
```
nit: a bit more obvious IMO

Reply:
Done. I'll keep the short version as `-f` for lack of a better alternative.